ilc/client.js

```diff
@@ -29,17 +32,26 @@ const appErrorHandlerFactory = (appName, slotName) => {
     return fragmentErrorHandlerFactory(registryConf, router.getCurrentRoute, appName, slotName);
 };
 
+const pluginManager = new PluginManager(require.context('./node_modules', true, /ilc-plugin-.*\/dist\/browser\.js$/));
```

Comment:
I have decided to use [`require.context`](https://webpack.js.org/guides/dependency-management/#requirecontext) to find every plugin and include them into `client.js` bundle
In this way, we can use common shared logic for PluginManager because the only thing that we have to change on the server side is to use require.context as a npm package